composer.json

```diff
@@ -24,5 +24,10 @@
             ]
         }
     },
+    "autoload": {
+        "psr-4": {
+            "Zaver\\": "src/"
```

Comment:
I would suggest adding a additional part to the namespace here for Krokedil. So it would be `Krokedil\\Zaver\\` instead. That way the convention is maintained, and we also dont add any potential issues with adding a PSR-4 autoload reference to namespaces that are not actually in the src folder, since the classes in the plugin that exist since before already are under the Zaver namespace.